lua/filetype/init.lua

```diff
@@ -227,7 +227,9 @@ function M.resolve()
     -- and set the filetype to that
     local shebang = analyze_shebang()
     if shebang then
-        shebang = builtin_map.shebang[shebang] or shebang
+        shebang = custom_map.shebang[shebang]
+            or builtin_map.shebang[shebang]
```

Comment:
`builtin_map` was removed entirely, so indexing it will cause an error.

Reply:
Yes, I'll update the PR with c712344caedfe443685259b84c6fe6b37b2c55f9; just wanted to ask first in case you were still working on it.